<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Файловая база данных</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableDB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="wordInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="translateInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="transcriptionInput">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>440</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>480</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backupButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Бекап</string>
    </property>
   </widget>
   <widget class="QPushButton" name="importButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Импорт</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Слово</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Перевод</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>320</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Транскрипция</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>520</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteByWord">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить слово</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteByTranslate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>480</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить перевод</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteByTranscription">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>520</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить транскрипцию</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="deleteInputWord">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>440</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="deleteInputTranslation">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="deleteInputTranscription">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteByID">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить ID</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="deleteInputID">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить БД</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchWord">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>440</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchTranslation">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>480</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchTranscription">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>520</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchID">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearSearch">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>400</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Отменить поиск</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
